### DISCUSSION TITLE
Students are unable to accept the assignment in github classroom

### DISCUSSION THREAD

**Discussion by bucsab12** [community/community#108068] [GitHub Education]
### Hi everyone,

I have seen this issue was brought up in the past by multiple users, but no solution was suggested, so I am opening this thread.

I created a git repository with an assignment as a part of github classroom. I sent the assignment link to several people but all of them told me that they were not able to accept the assignment. They encountered the following error:

"There was an error accepting the assignment. We were not able to add the user to your assignment, please try again.. If this problem persists, please have the Classroom admin contact support."

Users were able to accept the classroom in the past, up till several days ago.

I also created another classroom and reused the same assignment and tried to share that link, but the students ran into the same error.

I would greatly appreciate your help.
Thank you very much in advance.

**Comment by ChanvithaPraveen** [comment]
Yes, this happened to my Computer Vision Lecturer as well. He opened an assignment and instructed to students to commit and submit the assignment. But we couldn't submit it before the deadline because of this issue. I also think this may be an issue in sometimes. Would like to know about the reasons. Thanks.

**Reply by bucsab12** [reply]
Thank you for the reply. I hope someone will be able to let us know what is the cause for this issue. If it is just a glitch on github's side, or something else, since it worked normally a couple of days ago.

**Reply by ChanvithaPraveen** [reply]
""Students having trouble joining your GitHub Classroom assignment? It's likely a permission, setting, or template issue. Double-check student access, ensure repository visibility is correct (no "Internal"!), address template access if applicable, and have students clear browser cache. If stuck, contact GitHub Support. Keep students informed and offer alternative access like direct repository sharing. Patience and clear communication are key! Remember, you've got this!""

This above message was published by my batch representative that he got from the lab instructor. I think this would be the solution for me as well. When I try it with another browser it worked. Anyway the deadline was extended and already submitted it today. 

Hope this will clear you, Thank you very much!
